pkg/cloudprovider/aws/instance.go

```diff
@@ -304,3 +385,7 @@ func combineReservations(reservations []*ec2.Reservation) []*ec2.Instance {
 	}
 	return instances
 }
+
+func createIceCacheKey(instanceType string, capacityType string) string {
+	return fmt.Sprintf("%s-%s", instanceType, capacityType)
```

Comment:
I think od == on-demand, so dash feels a bit odd.
```suggestion
	return fmt.Sprintf("%s/%s", instanceType, capacityType)
```